src/drivers/driver.ts

```diff
@@ -1139,13 +1173,23 @@ export abstract class CMakeDriver implements vscode.Disposable {
             async (value: string) => expand.expandString(value, {...opts, envOverride: expanded_configure_env}));
         expanded_flags = await Promise.all(expanded_flags_promises);
       }
-      log.trace(localize('cmake.flags.are', 'CMake flags are {0}', JSON.stringify(expanded_flags)));
+      if (!usingCachedConfiguration) {
+        log.trace(localize('cmake.flags.are', 'CMake flags are {0}', JSON.stringify(expanded_flags)));
+      }
 
       // A more complete round of expansions
       await this._refreshExpansions();
 
       const timeStart: number = new Date().getTime();
-      const retc = await this.doConfigure(expanded_flags, consumer);
+      let retc: number;
+      if (usingCachedConfiguration) {
+        retc = await this.doCacheConfigure();
+        this._isConfiguredAtLeastOnce = true;
```

Comment:
```ts
this._isConfiguredAtLeastOnce = true;
```
Since you set this the same in both branches of the code, you can move this outside of the if/else.